### DISCUSSION TITLE
/feature/advance/visitor.html

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#6] [Announcements]
# /feature/advance/visitor.html

VanBlog 的官方网站

https://vanblog.mereith.com/feature/advance/visitor.html

**Comment by mosuzi** [comment]
就冲着这一趴不使用外部工具来实现，给你点个赞(๑•̀ㅂ•́)و✧

**Reply by Mereithhh** [reply]
一开始考虑过用 waline 内置的浏览量统计，但是不太好用也不好和后台结合起来，所以就自己写了233
